src/perf/shader.md

```diff
@@ -1,33 +1,45 @@
 ---
-title: Reduce shader compilation jank on mobile
+title: Shader compilation jank
 short-title: Shader jank
 description: What is shader jank and how to minimize it.
 ---
 
 {% include docs/performance.md %}
 
-If the animations on your mobile app appear to be janky,
-but only on the first run, you can _warm up_ the
-shader captured in the Skia Shader Language (SkSL) for a
-significant improvement.
-
-![Side-by-side screenshots of janky mobile app next to non-janky app]({{site.url}}/assets/images/docs/perf/render/shader-jank.gif)
+If the animations on your mobile app appear to be janky, but only on the first
+run, this is likely due to shader compilation. Flutter's long term solution to
+shader compilation jank is [Impeller][], which is in developer preview on master
+channel for iOS. Before continuing with the instructions below, please try out
+Impeller on iOS, and let us know in a GitHub issue if it doesn't address your
+issue.
```

Comment:
If we're not ready to encourage people to try it on Android, should we at least mention that we expect to apply a similar solution for Android?

Reply:
Done.